File: Beecrowd/Beginner/1435 - 1985/1962 - A Long, Long Time Ago/C/1962_ALongLongTimeAgo.c
#include <stdio.h>

int main() {
    int testCases;
    scanf("%d", &testCases);

    while (testCases--) {
        int yearsPassed;
        scanf("%d", &yearsPassed);

        if (yearsPassed < 2015) {
            printf("%d D.C.\n", 2015 - yearsPassed);
        } else printf("%d A.C.\n", yearsPassed - 2014);
    }

    return 0;
}

File: Beecrowd/Beginner/1435 - 1985/1962 - A Long, Long Time Ago/Python/1962_ALongLongTimeAgo.py
testCases = int(input())

for _ in range(testCases):
    yearsPassed = int(input())

    if yearsPassed < 2015:
        print(f"{2015 - yearsPassed} D.C.")
    else:
        print(f"{yearsPassed - 2014} A.C.")
